recipes/binutils/all/conanfile.py

```diff
@@ -1,15 +1,15 @@
-from conan import ConanFile
-from conan.tools.gnu import Autotools, AutotoolsToolchain
-from conan.tools.files import get, rmdir, rm, copy, apply_conandata_patches, export_conandata_patches
-from conan.errors import ConanInvalidConfiguration
-from conan.tools.microsoft import is_msvc
-from conan.tools.layout import basic_layout
-
 import os
 import re
 import typing
 import unittest
 
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rm, rmdir
+from conan.tools.gnu import Autotools, AutotoolsToolchain
+from conan.tools.layout import basic_layout
+from conan.tools.microsoft import is_msvc
 
 required_conan_version = ">=1.52.0"
```

Comment:
```suggestion
required_conan_version = ">=1.54.0"
```

1.53 for rm_safe, 1.54 for autotools install on Windows